src/main/java/com/celements/model/classes/fields/list/AccessRightLevelsField.java

```diff
@@ -0,0 +1,39 @@
+package com.celements.model.classes.fields.list;
+
+import javax.annotation.concurrent.Immutable;
+import javax.validation.constraints.NotNull;
+
+import com.xpn.xwiki.objects.classes.LevelsClass;
+import com.xpn.xwiki.objects.classes.ListClass;
+
+@Immutable
+public final class AccessRightLevelsField extends StringListField {
```

Comment:
Extend _EnumListField\<EAccessLevel>_ and overwrite _getListClass()_ and if needed also _serialize(List\<EAccessLevel>)_ and _resolveList(List\<?>)_